### DISCUSSION TITLE
127 version for CefSharp.Wpf.HwndHost

### DISCUSSION THREAD

**Discussion by SLT-World** [cefsharp/CefSharp.Wpf.HwndHost#26] [General]
Hello, could I know when will Chromium 127 for CefSharp.Wpf.HwndHost come out? It seems that the Nuget package CefSharp.Wpf already has Chromium 127, but CefSharp.Wpf.HwndHost is still stuck on Chromium 126. Thanks.

**Comment by amaitland** [comment]
You can subscribe to release only notifications here on github to be notified when there's a release.

It's unlikely I'll have time to get to this before next weekend. If you'd like to speed up the process and help out then by all means submit a PR.

Example of update commit https://github.com/cefsharp/CefSharp.Wpf.HwndHost/commit/be8cbf6b820662eb1dd5b4e0b6335764d99a52a3

Just need to make a minor change for https://github.com/cefsharp/CefSharp/commit/57f8cd344f63dc51bd6633975ebf418d0b7cbdea#diff-d536f95a376210319dd04246736385251bb23a5a1c40d8baa5a37f6a129505d4R512

**Comment by SLT-World** [comment]
Thanks for the guide, no rush, I just submitted a PR (https://github.com/cefsharp/CefSharp.Wpf.HwndHost/pull/27) with the needed changes.

**Reply by amaitland** [reply]
Thanks 👍 Minor style tweak required, other than that looks great 👍

**Reply by amaitland** [reply]
New version has been pushed, should arrive shortly.

**Reply by SLT-World** [reply]
Thanks for the new version.

**Reply by amaitland** [reply]
Thanks for the help

**Comment by SLT-World** [comment]
Hi again, I realized that the CefSettings here was missing `disable-chrome-login-prompt` and `hide-crash-restore-bubble` meanwhile the others have it. I should have seen this in the past PR 😓. I've created a new PR (https://github.com/cefsharp/CefSharp.Wpf.HwndHost/pull/28) adding these flags.

This isn't a very serious issue so maybe the changes can be published later in 128.

`ChromeRuntime = true` support will be removed in 128 so I don't think adding that is necessary, spares some work removing it in 128 too.
